bin/run-standard-tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -eo pipefail
+
+display_and_run() {
+    echo "***" "$@"
+	eval $(printf '%q ' "$@")
+}
+
+# reset workdir to state from git (to remove possible rewritten dependencies)
+display_and_run git reset --hard
+
+# Fmt check
+echo "*** go fmt ./..."
+find . -name '*.go' -not -path "./.gopath/*" -not -path "./vendor/*" -exec gofmt -l {} + > go-fmt.out
```

Comment:
:hairy_eyebrow:

`./...` is... a thing...

Reply:
right but it's purposely avoiding including vendor files and .gopath files. because .gopath stuff fails.